src/main/java/io/stargate/sgv2/jsonapi/config/constants/HttpConstants.java

```diff
@@ -15,9 +15,27 @@ public interface HttpConstants {
   /** JSON API Embedding serive Authentication token header name. */
   String EMBEDDING_AUTHENTICATION_TOKEN_HEADER_NAME = "x-embedding-api-key";
 
+  /** JSON API Embedding serive access id header name. */
+  String EMBEDDING_AUTHENTICATION_ACCESS_ID_HEADER_NAME = "x-embedding-access-Id";
+
+  /** JSON API Embedding serive secret id header name. */
+  String EMBEDDING_AUTHENTICATION_SECRET_ID_HEADER_NAME = "x-embedding-secret-Id";
+
   /**
-   * @return Embedding service header name <code>20</code>.
+   * @return Embedding service header name for token<code>20</code>.
```

Comment:
I know it was already there but.... what does "20" here mean?

Reply:
Copy/paste issue. Resolved.